Source/WebCore/platform/mediastream/gstreamer/RealtimeIncomingAudioSourceGStreamer.cpp

```diff
@@ -25,12 +25,18 @@
 #include "GStreamerAudioData.h"
 #include "GStreamerAudioStreamDescription.h"
 
+GST_DEBUG_CATEGORY_EXTERN(webkit_webrtc_endpoint_debug);
+#define GST_CAT_DEFAULT webkit_webrtc_endpoint_debug
+
 namespace WebCore {
 
 RealtimeIncomingAudioSourceGStreamer::RealtimeIncomingAudioSourceGStreamer(AtomString&& audioTrackId)
     : RealtimeMediaSource(RealtimeMediaSource::Type::Audio, WTFMove(audioTrackId))
     , RealtimeIncomingSourceGStreamer()
 {
+    static uint64_t sourceCounter = 0;
```

Comment:
What thread creates this?

Reply:
Hm good question! From a quick review of call sites, I'd say this isn't thread-safe indeed. I'll wrap this in a `Atomic`.